### DISCUSSION TITLE
Remove header and footer on dompdf

### DISCUSSION THREAD

**Discussion by Nijat-Majidli** [dompdf/dompdf#3003] [General]
By using Dompdf library, I created a pdf file from twig template. This pdf file also contains header part of twig template. 
How can I remove header part from pdf file ?
Can anybody help me?

**Comment by bsweeney** [comment]
Presumably the header is identified by a particular HTML class name or id. If this is the case you can use CSS media targeting to hide the header/footer only when rendered by Dompdf. In addition to the standard media types Dompdf supports the "dompdf" media type, which should be ignored by other user agents.

So, for example, if you header is defined as follows:
```
<div id="header">
  <!-- header content -->
</div>
```

You can hide it when rendered by Dompdf adding the following CSS to your stylesheet:
```
@media dompdf {
  #header {
    display: none;
  }
}
```

I also imagine it's possible to define an alternate twig layout when rendering to PDF, but that's outside the realm of "how to do this with Dompdf."
